tests/cli/test_plot.py

```diff
@@ -62,3 +117,33 @@ def test_plot_dissociation_curve(run_cli_command, generate_dissociation_curve_no
     options = [str(node.pk), '--print-table', '--precisions'] + [str(p) for p in precisions]
     result = run_cli_command(plot.cmd_plot_dissociation_curve, options)
     data_regression.check({'output_lines': result.output_lines})
+
+
+@pytest.mark.usefixtures('aiida_profile')
+def test_plot_dissociation_curve_output_file(run_cli_command, generate_dissociation_curve_node, monkeypatch, tmp_path):
+    """Test the `plot_dissociation_curve` command with the `--output-file` option."""
+    from aiida_common_workflows.common.visualization import dissociation
+
+    def get_dissociation_plot(_, __):
+        return plt
+
+    monkeypatch.setattr(dissociation, 'get_dissociation_plot', get_dissociation_plot)
+
+    node = generate_dissociation_curve_node().store()
+    output_file = tmp_path / 'plot.png'
+    options = [str(node.pk), '--output-file', str(output_file)]
+
+    result = run_cli_command(plot.cmd_plot_dissociation_curve, options)
```

Comment:
Would maybe add a check that the file was actually written to disk. Same goes for the table option. There you could even check the content, but don't think that is crucial